tpm2tools/handles_test.go

```diff
@@ -0,0 +1,34 @@
+package tpm2tools
+
+import (
+	"testing"
+
+	"github.com/google/go-tpm-tools/simulator"
+	"github.com/google/go-tpm-tools/tpm2tools/tpm2toolstest"
+	"github.com/google/go-tpm/tpm2"
+)
+
+const (
+	// Maximum number of handles to keys tests can create within a simulator.
+	maxHandles = 3
+)
+
+func TestHandles(t *testing.T) {
+	simulator, err := simulator.Get()
```

Comment:
Use `internal.GetTPM()` see [this example](https://github.com/google/go-tpm-tools/blob/551cd9014fbdf21ed8bbe3e1a5cb7ec1564c3c2d/tpm2tools/keys_test.go#L30-L32)

Reply:
Done.